pkg/providers/imagefamily/bootstrap/cse_cmd.sh.gtpl

```diff
@@ -144,4 +144,6 @@ KUBE_CA_CRT="{{.KubeCACrt}}"
 KUBENET_TEMPLATE="{{.KubenetTemplate}}"
 CONTAINERD_CONFIG_CONTENT="{{.ContainerdConfigContent}}"
 IS_KATA="{{.IsKata}}"
+ENABLE_IMDS_RESTRICTION=false
+INSERT_IMDS_RESTRICTION_RULE_TO_MANGLE_TABLE=false
```

Comment:
Why set the defaults here like this rather than in NodeBootstrappingVariables like we have previously?

Reply:
Overall we want to be shrinking the required contract. There is no need to propagate a temporary fix (which would like to remove as soon as the contract allows it) higher up the data flow chain than absolutely necessary. When and if we decide to actually expose this for control / customization - then would be the time to establish the correct data flow. Yes, this is a bit of an inconsistency with other settings; we may want to start taking similar approach to them as well - though maybe let's wait until the contract moves forward.